ci/azure-steps.yml

```diff
@@ -38,28 +38,16 @@ steps:
      DownloadFile -Source 'https://github.com/mesonbuild/cidata/raw/master/win32/pkg-config.exe' -Destination $(System.WorkFolder)\pkg-config.exe
      DownloadFile -Source 'https://download.microsoft.com/download/D/B/B/DBB64BA1-7B51-43DB-8BF1-D1FB45EACF7A/msmpisdk.msi' -Destination msmpisdk.msi
      DownloadFile -Source 'https://download.microsoft.com/download/D/B/B/DBB64BA1-7B51-43DB-8BF1-D1FB45EACF7A/MSMpiSetup.exe' -Destination MSMpiSetup.exe
-     if ($env:compiler -ne 'msvc2015') {
-       Start-Process msiexec.exe -ArgumentList '/i msmpisdk.msi /quiet' -Wait
```

Comment:
The code inside this conditional should stay, since the condition is `-ne 'msvc2015'`?